models/DataObject/ClassDefinition/Data/Extension/Relation.php

```diff
@@ -22,9 +22,9 @@
  * @method bool getDocumentsAllowed()
  * @method bool getAssetsAllowed()
  * @method bool getObjectsAllowed()
- * @method string[] getDocumentTypes()
- * @method string[] getAssetTypes()
- * @method string[] getClasses()
+ * @method array getDocumentTypes()
```

Comment:
why?

Reply:
The return value is a multidimensional array. See https://github.com/pimcore/pimcore/blob/b34fdfafd2bd703ffe7a6e8f6ca80aceaee3f4c1/models/DataObject/ClassDefinition/Data/Extension/Relation.php#L44-L53

Reply:
yep, $strArray = $asArray ? '[]' : ''; makes the difference